src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -133,6 +133,11 @@ import { isStandalone } from 'vs/base/browser/browser';
 				'description': nls.localize('enablePreviewFromQuickOpen', "Controls whether editors opened from Quick Open show as preview. Preview editors do not keep open and are reused until explicitly set to be kept open (e.g. via double click or editing)."),
 				'default': false
 			},
+			'workbench.editor.enablePreviewFromCodeNavigation': {
+				'type': 'boolean',
+				'description': nls.localize('enablePreviewFromCodeNavigation', "Controls whether preview editors should be kept open as regular editors when opening a second editor from the preview (e.g. via go to definition)"),
+				'default': true
```

Comment:
```suggestion
				'default': false
```

Reply:
Are you sure? In my implementation `true` means new behavior (keeping old preview editor open). Your suggestion would make the old behavior default again.  I can also change it to `false` means new behavior, if that's what you mean.